src/diff/amfGraphDifferencer.ts

```diff
@@ -269,13 +284,17 @@ export function parseNodeDelta(
   delta: AmfGraphDeltaTypes.ArrayElementDelta<AmfGraphTypes.Node>,
   deltaType: DeltaType
 ): NodeChanges {
```

Comment:
```suggestion
): NodeChanges | undefined {
```
The return type should reflect the returned values.